proto/user_stats/user_stats.proto

```
@@ -0,0 +1,65 @@
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+
+package user_stats;
```

Buf (1.57.2)

Files with package "user_stats" must be within a directory "user_stats" relative to root but were in directory "proto/user_stats".

(PACKAGE_DIRECTORY_MATCH)